transit_odp/avl/client/cavl.py

```diff
@@ -166,3 +168,100 @@ def validate_feed(
                 created=timezone.now(),
                 version=siri_version,
             )
+
+
+class CAVLSubscriptionService(ICAVLSubscriptionService):
+    AVL_CONSUMER_URL = settings.AVL_CONSUMER_API_BASE_URL
+
+    @staticmethod
+    def _get_error_response(exception: RequestException) -> str:
+        return (
+            exception.response.json()
+            if hasattr(exception.response, "json")
+            else "(empty)"
+        )
+
+    def subscribe(
```

Comment:
Do we need some tests for this function? Similar to what's in `transit_odp/avl/client/tests/test_cavl_client.py`